home/tests/test_logout.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+@pytest.mark.django_db
+class TestLogout:
+    def test_status_code(self, client):
+        response = client.get('/logout/')
+        assert response.status_code == 302  # redirected
+
+    def test_redirection_default_url(self, client):
+        response = client.get('/logout/')
+        assert response.url == '/'  # redirected to home page.
+
+    def test_logout(self, client):
+        client.login(username="Lior", password="LiorLior")
+        response = client.get('/')
+        assert response.wsgi_request.user.is_authenticated  # successfully logged in
+        response = client.get('/logout/')
+        assert not response.wsgi_request.user.is_authenticated   # successfully logged out
+
+    def test_unauthenticated_user_view(self, client):
+        response = client.get('/')
+        assert "Login" in str(response.content)
+        assert "Logout" not in str(response.content)
+
+    def test_redirection_url(self, client, request):
+        # each page transfer his own url as a next parameter
+        response = client.get('/logout/?next=/about/')
+        assert response.url == '/about/'  # redirected to home page.
```

Comment:
the comment is lying, why do you need it anyway?

Reply:
oops